wren-ai-service/src/pipelines/common.py

```diff
@@ -494,7 +494,9 @@ def dry_run_pipeline(pipeline_cls: BasicPipeline, pipeline_name: str, **kwargs):
     from src.config import settings
     from src.core.pipeline import async_validate
     from src.providers import generate_components
-    from src.utils import init_langfuse
+    from src.utils import init_langfuse, setup_custom_logger
+
+    setup_custom_logger("wren-ai-service", level_str=settings.logging_level)
```

Comment:
why do we need setup_custom_logger here?

Reply:
it will be more easy to see the log and adjust the level in using dry run pipeline for testing